lib/bambora/rest/client.rb

```diff
@@ -40,6 +40,8 @@ def delete(path:, headers:)
 
       def connection
         @connection ||= Faraday.new(url: base_url) do |faraday|
+          faraday.request :multipart
+          faraday.request :url_encoded
```

Comment:
spent some time figuring out this defaults all request to this content-type :) seems those 2 are fallback values.

Reply:
Ah, yeah, this isn't immediately clear in Faraday, but the middleware stack matches mime types and only executes if there's a match.

https://github.com/lostisland/faraday/blob/9865ebbf8ffdbb2f12bb3af7f02baa10070e8c5a/lib/faraday/request/url_encoded.rb#L18